### PR TITLE
NIJ update data lists

### PR DESCRIPTION
**Pull request by davet1985**

### CODE REVIEW

**Comment by jackmaloney** [comment]
Why do we have a list of stations and ports on self serve?

File: test/lib/flight-lookup.spec.js [hunk 1]
```diff
@@ -160,12 +160,19 @@ describe('lib/flight-lookup', function() {
                     key: 'code',
                     val: 'KWI'
                 };
+                /*
```
Comment (jackmaloney) [inline test/lib/flight-lookup.spec.js]: Commented out code

Comment (davet1985) [inline test/lib/flight-lookup.spec.js]: 😢

**Comment by davet1985** [comment]
Stations and ports on self-serve are the future. Aren't they?

**Pull request merged**